analysis/test/src/pointers.rs

```diff
@@ -340,6 +343,7 @@ pub unsafe extern "C" fn test_arg() {
     let mut s = malloc(::std::mem::size_of::<S>() as libc::c_ulong);
     foo(s);
     let t = s;
+    free(s as *mut libc::c_void);
```

Comment:
Should `t` be freed instead of `s`?

Reply:
if they're the same address, does it make a difference?

Reply:
there was no particular intention, so this is fine